apps/worker/src/app/workflow/usecases/send-message/send-message-push.usecase.ts

```diff
@@ -147,7 +149,17 @@ export class SendMessagePush extends SendMessageBase {
 
       const overrides = command.overrides[integration.providerId] || {};
 
-      await this.sendMessage(integration, channel.credentials.deviceTokens, title, content, command, data, overrides);
+      await this.sendMessage(
+        subscriber,
+        integration,
+        channel.credentials.deviceTokens,
+        title,
+        content,
+        command,
+        command.payload,
```

Comment:
this line basically is the fix... the payload was containing additional data that shouldn't be sent with the FCM data message